server/routes/search.js

```diff
@@ -6,34 +8,50 @@ var searchRoutes = function(router) {
 
     var db = req.db,
         collection = db.get('ControlTweets'),
-        features = [];
+        features = [],
+        q = req.body.tweetText.split(','),
+        queries = [];
 
-    //execute db query
-    collection.find({
-      $text: {
-        $search: req.body.tweetText
-      }
-    }, { tln: 1, tlt: 1 }, function(err, cursor) {
+    // multiple find statements for search multiple keywords
+    q.forEach(function(v, i, a){
```

Comment:
I probably should eventually find time and create proper eslint file. But could you keep a space between `) {`